src/nodes/chunk_append/planner.c

```diff
@@ -401,10 +401,20 @@ ts_chunk_append_get_scan_plan(Plan *plan)
 			else
 				return NULL;
 			break;
-		case T_MergeAppend:
 		case T_Agg:
-			return NULL;
+			if (plan->lefttree != NULL && IsA(plan->lefttree, CustomScan))
+			{
+				Assert(plan->righttree == NULL);
+				Assert(castNode(CustomScan, plan->lefttree)->scan.scanrelid > 0);
+				return (Scan *) plan->lefttree;
+			}
+			else
+			{
+				return NULL;
+			}
```

Comment:
```suggestion

			return NULL;
```
Nit. For simplicity!!!

Reply:
Thanks, changed.